### PR TITLE
Sentry: Filter '/healthz' transactions + Enable in API v2

### PR DESCRIPTION
**Pull request by ouziel-slama**

### CODE REVIEW

File: counterparty-core/counterpartycore/lib/api/api_server.py [hunk 1]
```diff
@@ -244,6 +255,8 @@ def handle_route(**kwargs):
 
 
 def run_api_server(args):
+    logger.info("Starting API Server v2.")
```
Comment (adamkrellenstein) [inline counterparty-core/counterpartycore/lib/api/api_server.py:258]: I think we shouldn't specify the version here. It's just the default now :)

File: counterparty-core/counterpartycore/lib/sentry.py [hunk 2]
```diff
@@ -29,6 +31,17 @@ def before_send(event, _hint):
     return event
 
 
+def filter_transactions(event, _hint):
```
Comment (adamkrellenstein) [inline counterparty-core/counterpartycore/lib/sentry.py:34]: Do we need to filter client-side? Sentry says it'll do it server-side.

**Review by adamkrellenstein — APPROVED** [review]

File: counterparty-core/counterpartycore/lib/api/api_server.py (hunk 1 above)
Comment (ouziel-slama) [reply counterparty-core/counterpartycore/lib/api/api_server.py:258]: :-) yes!

Comment (ouziel-slama) [reply counterparty-core/counterpartycore/lib/api/api_server.py]: done

File: counterparty-core/counterpartycore/lib/sentry.py (hunk 2 above)
Comment (ouziel-slama) [reply counterparty-core/counterpartycore/lib/sentry.py:34]: yes we could just rename the transaction with a filtered name but for bandwith usage perhaps better to filter here ?

Comment (ouziel-slama) [reply counterparty-core/counterpartycore/lib/sentry.py]: I renamed the routes to "healthcheck", so should be filtered server side, and removed the filter client side.

Comment (adamkrellenstein) [reply counterparty-core/counterpartycore/lib/sentry.py]: eh I think it's a tiny amount of bandwidth, and it's only for servers we host ourselves, really. Also if we don't filter client-side, we can turn the filter off server-side later if we want.

**Pull request merged**